portfolio/src/main/webapp/script.js

```diff
@@ -82,6 +89,35 @@ function renderListComments({firstName, lastName, commentText, date}) {
   document.getElementById('comments').appendChild(content);
 }
 
+function createMap() {
+  const map1 = new google.maps.Map(document.getElementById('map'), {
```

Comment:
I don't think this would conflict with anything if it was just called `map`

Reply:
I initially had different maps and later deleted them but forgot to change the name. Changes made.